packages/muelu/src/Graph/MatrixTransformation/MueLu_CoalesceDropFactory_kokkos_def.hpp

```diff
@@ -233,6 +260,83 @@ std::tuple<GlobalOrdinal, typename MueLu::LWGraph_kokkos<LocalOrdinal, GlobalOrd
   if (((algo == "classical") && (classicalAlgoStr.find("scaled") != std::string::npos)) || ((algo == "distance laplacian") && (distanceLaplacianAlgoStr.find("scaled") != std::string::npos)))
     TEUCHOS_TEST_FOR_EXCEPTION(threshold > 1.0, Exceptions::RuntimeError, "For cut-drop algorithms, \"aggregation: drop tol\" = " << threshold << ", needs to be <= 1.0");
 
+  //////////////////////////////////////////////////////////////////////
+  // translate to new options
+
+  bool useBlocking                          = false;
+  std::string socUsesMatrix                 = "A";
+  std::set<std::string> validSocUsesMatrix  = {"A", "distance laplacian"};
+  std::string socUsesMeasure                = "sa";
+  std::set<std::string> validSocUsesMeasure = {"unscaled", "sa", "signed sa", "signed rs"};
```

Comment:
We might consider to spell out abbreviations such as "sa" or "rs" to make the parameter value self-explaining and not requiring additional documentation.